doc/README.rst

```diff
@@ -117,7 +117,7 @@ Sphinx supports easy customization of the generated documentation
 appearance through the use of themes.  Replace the theme files and do
 another ``make htmldocs`` and the output layout and style is changed.
 The ``read-the-docs`` theme is installed as part of the
-``requirements.txt`` list above, and will be used if it's available, for
+``requirements-doc.txt`` list above, and will be used if it's available, for
```

Comment:
There's no mention of using pip3 install -r requirements-doc.txt above; there should be if this PR is going to go in